app/sagas/createChannel.js

```diff
@@ -81,6 +81,7 @@ const handleSuccess = function* handleSuccess({ data }) {
 	if (isMasterDetail) {
```

Comment:
If you're doing lots of things inside `goRoom` when isMasterDetail, why do we need this still?
Same for line 84.
If we're unifying the usage, why do we need to deal with edge cases?